airflow/configuration.py

```diff
@@ -41,6 +41,22 @@ def expand_env_var(env_var):
         else:
             env_var = interpolated
 
+def run_bash_command(command):
```

Comment:
thinking of how best to resolve this dependency but a case can be made for keeping this here. there are several other places in code where bash and other types of command are run using subprocess inline. thoughts?